source/manual/dns.html.md

```diff
@@ -35,25 +35,75 @@ Note that we __do not__ manage any other DNS records: if you get a request conce
 When you've verified the authenticity of the request as per the SRE docs above, you should:
 
 1. Make the changes in govuk-dns-config (see [example](https://github.com/alphagov/govuk-dns-config/pull/851))
-1. Deploy the changes via the [Deploy_DNS Jenkins job](https://deploy.blue.production.govuk.digital/job/Deploy_DNS/). You'll need production admin access to do this.
+1. Use the gds-cli to deploy the changes via the [Deploy_DNS Jenkins job][]
 
-Deploying is a multi-step process:
+Before you start make sure your machine is set up so you can access AWS and [Google Cloud Platform (GCP)][] using command line tools.
+
+The deployment process is complex, so it's much easier to [use the automation provided by the GDS CLI](#running-govuk-dns-using-gds-cli), although it [can be done manually](#running-govuk-dns-manually-using-jenkins).
+
+### Running govuk-dns using gds-cli
+
+gds-cli provides a command which automates gathering the required credentials and triggering Jenkins. The command authenticates to Jenkins using GitHub, so it expects `GITHUB_USERNAME` / `GITHUB_TOKEN` environment variables to be set. It calls out to the `aws` and `gcloud` command line tools to fetch the other required credentials.
+
+You can get the relevant help for your version of gds-cli with:
+
+```
+$ gds govuk dns --help
+NAME:
+   gds govuk dns - deploy DNS via Jenkins
+
+USAGE:
+   gds govuk dns [command options] [arguments...]
+
+OPTIONS:
+   --provider value, -p value            DNS provider to use
+   --zone value, -z value                DNS zone for
+   --action value, -a value              action to apply. e.g. plan
+   --github-username value, -u value     GitHub username used to log into Jenkins, can be set via env variable GITHUB_USERNAME [$GITHUB_USERNAME]
+   --role value, -r value                Name of the AWS role used to get credentials
+   --assume-role-ttl value, --art value  Expiration time for the assumed role. Most roles are configured to permit up to 60m. (default: "60m")
+   --skip-mfa, --sm                      Don't ask for MFA (default: false)
+   --help, -h                            show help (default: false)
```

Comment:
I think an example of running the command would suffice as otherwise this would go out of date when the tool gets updated.

Reply:
I've gone for a half way house and removed the OPTIONS section (the rest of the help should be reasonably stable, and somehow I find its presence here reassuring).